.github/workflows/tests.yml

```diff
@@ -155,7 +155,7 @@ jobs:
         if: always()
         with:
           name: playwright-report
-          path: playwright-report/
+          path: ./test-apps/e2e/test-app-0/playwright-report/
```

Comment:
Mh, I think the names of the `test-app` folders are dynamic and therefore we would need to find a way to upload reports from `test-app-*`.

Reply:
Sorry, I wasn't clear, but that's basically my question. Right now we can rely on it being test-app-0 simply because we know we reset the instance and it's always going to be test-app-0, but there are a lot of things that could change in the future and cause it to be a higher number.

Reply:
We could force e2e tests to run with a specified path and use that, that's the only option I can think of

Reply:
Alternatively, the test script could end by copying the results to a fixed path, but I really don't like that idea because it makes it even more fragile. However, it would allow multiple test apps within a single instance to upload their results, which I will need in a few months when we add data transfer e2e tests.

So essentially, I'm wondering if this hack is good enough for now, and then we revisit the idea later to make it more robust.

Reply:
Could we use `outputFile` for a playwright reporter to store the report outside of the test app?